From: Sascha Hauer <s.hauer@pengutronix.de>
Subject: Add hack for default partition

Let grub find the default partition the rootfs is installed
on. There must be a better (proper) way to archieve this,
but I don't know any.

Signed-off-by: (not for upstream)
---
Index: grub-0.97/stage2/asm.S
===================================================================
--- grub-0.97.orig/stage2/asm.S
+++ grub-0.97/stage2/asm.S
@@ -86,7 +86,7 @@ ENTRY(main)
 	. = EXT_C(main) + 0x8
 
 VARIABLE(install_partition)
-	.long	0xFFFFFF
+	.long	0x00FFFF
 /* This variable is here only because of a historical reason.  */
 VARIABLE(saved_entryno)
 	.long	0
